### ISSUE TITLE
Wrong annotation for Tags._fuzzy_search

### ISSUE THREAD

**Issue by F4zii** [python-discord/bot#789]
## Description
The current declaration of _fuzzy_search is:
```py
@staticmethod
def _fuzzy_search(search: str, target: str) -> int
```

while the return value is a float.
```py
return current / len(_search) * 100
```

## Available fixes

We can change the return value from **int** to **float** or convert the division mark to floor the result:

 ```py
 def _fuzzy_search(search: str, target: str) -> float
```
 
**or**

```py
return current // len(_search) * 100
```

If you work on this, consider working on #787

**Comment by F4zii** [comment]
I can fix it when it's approved :+1:

**Comment by sco1** [comment]
This would probably be best included with the fix for #787

**Comment by sco1** [comment]
Oh never mind you already said that

**Comment by MarkKoz** [comment]
@F4zii You can go ahead and work on this but since someone else was assigned to the other issue maybe they should take care of this one too.

**Comment by F4zii** [comment]
Ok, sure thing

**Comment by Numerlor** [comment]
This has been addressed in #803
